.travis.yml

```diff
@@ -0,0 +1,30 @@
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements.  See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+language: java
+notifications:
+  irc:
+    channels:
+      - "chat.freenode.net#accumulo"
+    use_notice: true
+    on_success: change
+    on_failure: always
+    template:
+      - "%{result} %{repository_slug} %{branch} (%{build_url}): %{message}"
+cache:
+  directories:
+    - $HOME/.m2
+jdk:
+  - oraclejdk8
+script: mvn clean verify
```

Comment:
This script needs `install: true` to skip the costly attempt by Travis CI which causes maven deps to be fetched twice. See main accumulo repo's `.travis.yml`